### PR TITLE
refactor: use v2 of dependabot

### PR DESCRIPTION
**Pull request by vinayakkulkarni**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [x] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

**Other information:**
- [x] Reference: https://github.com/vuejs/vuepress/pull/2162#issuecomment-661918383

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    time: '00:00'
+    timezone: Asia/Calcutta
```
Comment (billyyyyy3320) [inline .github/dependabot.yml:8]: The default reviewer is @bencodezen, Ben would you like to choose a proper time and timezone?

**Review by billyyyyy3320 — COMMENTED** [review]
LGTM

**Comment by vinayakkulkarni** [comment]
Sure.. Will update in some time.

File: .github/dependabot.yml (hunk 1 above)
Comment (bencodezen) [reply .github/dependabot.yml:8]: Yeah that would be helpful.

**Comment by vinayakkulkarni** [comment]
I've removed the `labels` setting cause `dependabot` automatically adds the labels as per the ecosystem:

for example:
Screenshot 2020-09-16 at 6 45 29 PM

Since we have two ecosystems that `dependabot` will look after, viz. JavaScript & GitHub actions, it'll automatically add
`javascript` & `github_actions` as two new labels along side `dependencies` label to each PR.

Isn't it cool?

**Pull request closed without merging**